packages/protocol/test/L1/TaikoL1.integration.test.ts

```diff
@@ -195,9 +199,10 @@ describe("integration:TaikoL1", function () {
             };
 
             const inputs = buildProposeBlockInputs(block, meta);
-
-            await expect(taikoL1.proposeBlock(inputs)).to.be.revertedWith(
-                "L1:placeholder"
+            await txShouldRevertWithCustomError(
+                taikoL1.proposeBlock(inputs, { gasLimit: 500000 }),
```

Comment:
Give a gas limit other wise the test will fail due to "unable to estimate gas limit"